map/src/context/RoutingManager.js

```diff
@@ -102,60 +103,439 @@ function filterMode(data) {
     return Object.fromEntries(Object.entries(data).filter(([key]) => !key.includes('rescuetrack')));
 }
 
+function osrmToFeaturesCollection(osrm) {
+/*
+    convert OSRM routes[] to features[] "LineString"
+    convert OSRM legs[].steps[] to features[] "Point" (OSRM's turns)
+    convert OSRM waypoints[] to features[] "Point" (OSRM's start-end)
+    convert OSRM duration/distance to properties
+
+    now support only 1st of routes[]
+*/
+    const features = [];
+
+    const cap = (s) => s && s[0].toUpperCase() + s.slice(1);
+
+    const maneuver = (s) => {
+        // imperative
+        const type = cap(s?.maneuver?.type ?? ''); // Turn
+        const modifier = cap(s?.maneuver?.modifier ?? ''); // Left
+
+        // target
+        const name = s?.name ?? ''; // street
+        const ref = s?.ref ? ` (${s.ref})` : ''; // (REF)
+
+        // go
+        const distance = s?.distance ?? '';
+
+        const imperative = type + ' ' + modifier;
+        const target = name ? 'to ' + name + ' ' + ref : '';
+        const go = distance > 0 ? 'and go ' + distance + ' meters' : '';
+
+        return `${imperative} ${target} ${go}`; // Turn Slight right to Schirmschneise and go 621 meters
```

Comment:
delete comment

Reply:
refused (slightly changed): this is example of string format which is returned by function; it's useful if you need to change format